common/authorization/authorizer.go

```diff
@@ -86,13 +89,21 @@ func NewPermission(permission string) Permission {
 	}
 }
 
+func (d domainData) Groups(groupType string) []string {
```

Comment:
I'm not sure if moving this method to elsewhere would add any benefit. "Groups" are not used elsewhere except in this, authorization related package. Maybe "domainData" name is a bit misleading, this can be named as "groupData".

On the other hand, if there's a need to add first-class "domain groups" support, then yeah, this should be moved to a dedicated, group management place. But for now, you have to use domain free-form metadata functionality to describe read/write groups and moving helper method out of here doesn't add any benefit IMHO.

Reply:
yeah, sorry, that was a discussion I was having internally. No concerns.